### PR TITLE
Fix #2510 - cancer case variants button names

### LINKED ISSUE #2510: Change "Clinical SNV/INDEL" to "Clinical somatic SNV/INDEL" for cancer view, same for SVs
Is it possible to add word "somatic" to "clinical SNV/INDEL" ?

The reason is we are not going to run MIP for normal sample to get the germline mutations. BASLAMIC already does it for targetseq and whole exome, and we'll do the same for WGS applications.

Thus we'd like to have both germline and somatic in the same view for each cancer case.

i.e. something like this:

image

So is this technically possible? I mean changing the wording in the cancer view. I'll open another issue for the germline part. (i mean if there is a validation for the texts and buttons and changing wording breaks anything elsewhere).

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality: rename cancer clinical variants button more clearly to indicate the cancer (somatic) and constitutional (germline) variants.

**How to test**:
1. add variants files to regular, cancer and research to a cancer track case e.g. in `scout/demo/cancer.load_config.yaml`
```
vcf_cancer: scout/demo/cancer_test.vcf.gz
vcf_cancer_sv: scout/demo/manta_vep_94_annotated_sv_cancer_file.vcf.gz

vcf_snv: scout/demo/cancer_test.vcf.gz
vcf_sv: scout/demo/manta_vep_94_annotated_sv_cancer_file.vcf.gz

vcf_snv_research: scout/demo/cancer_test.vcf.gz
vcf_sv_research: scout/demo/manta_vep_94_annotated_sv_cancer_file.vcf.gz

vcf_cancer_research: scout/demo/cancer_test.vcf.gz
vcf_cancer_sv_research: scout/demo/manta_vep_94_annotated_sv_cancer_file.vcf.gz
```
Also check an RD case to make sure things look as before there.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.
Screenshot 2021-06-01 at 09 11 28

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
After loading research vars to a cancer case: 🆗 

image


Checked RD case, after loading research as well: 🆗 

image

File: scout/server/blueprints/cases/templates/cases/case.html [hunk 1]
```diff
@@ -183,12 +183,17 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
   {% if case.is_research%}
   <div class="form-group">
     <div class="btn-group">
-    {% if case.track=="cancer" %}
-      <a class="btn btn-outline-dark" href="{{ url_for('variants.cancer_variants', institute_id=institute._id, case_name=case.display_name, variant_type='research') }}">Research SNV and INDELs</a>
-      <a class="btn btn-outline-dark" href="{{ url_for('variants.cancer_sv_variants', institute_id=institute._id, case_name=case.display_name, variant_type='research') }}">Research structural variants</a>
-    {% else %}
-      <a class="btn btn-outline-dark" href="{{ url_for('variants.variants', institute_id=institute._id, case_name=case.display_name, variant_type='research') }}">Research SNV and INDELs</a>
-      <a class="btn btn-outline-dark" href="{{ url_for('variants.sv_variants', institute_id=institute._id, case_name=case.display_name, variant_type='research') }}">Research structural variants</a>
+    {% if case.vcf_files.vcf_snv_research %}
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/case.html:186]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
It works perfectly!

**Comment by northwestwitch** [comment]
I'm in a merging spree. Can we merge this one? 🤔

**Comment by dnil** [comment]
It is good that we had the cancer discussion yesterday, and talked about having germline and somatic in different cases. I note that they will not load nicely together. Remember the avoid-loading-same-variant issue from last week? This has the same issue, but not not only SVs get the same id: variant_id should probably have had category in it as well...

**Pull request merged**